docs/mesh_inventory_examples.md

```diff
@@ -0,0 +1,60 @@
+# Automation Platform Mesh Inventory Examples
+<b><i>Note</b> - we use the term `mesh` in this document to describe a network comprising of vertices or `nodes`. Communication between nodes is defined by the term `edges`, or in computer terms, a connection on the transport layer such as `TCP`, `UDP` or `unix sockets`.</i>
+
+Please refer to [this](./receptor_mesh.md) document for a more in-depth explanation of the objects in the `mesh` network. 
+
+The examples below are an attempt to deliver `ansible` inventory files that reflect typical user environments. While we cannot encapsulate every single customer scenario, if you need a starting point to start building out your inventory file, please refer to the examples [below](##Example-Inventory-Files).
+
+| Node Type | Description |
+| --------- | ----------- |
+| Control | Nodes that run persistent AWX services, and delegate jobs to hybrid and execution nodes|
+| Hybrid | Nodes that run persistent AWX services and execute jobs from user-space|
+| Hop | Actionless nodes in a given network. They will have [receptor](./receptor_mesh.md) but remain actionless |
+| Execution | Nodes that run jobs delivered from Control-Nodes (jobs submitted from the user-space) |
+
+## Example Inventory Files
+--------------------------
+
+### Standard Control Plane (3 Node) and (n)-Execution Nodes
+![](./img/three_node_control_plane_n_execution_nodes.png)
+<details>
+    <summary>Code for Inventory File</summary>
+</details>
+
+### Standard Control plane and Execution Topology with Hop Nodes
```

Comment:
It might be good to have a bit of a "why" for these different cases. In this case, introducing hop nodes, obviously the user has nodes that are not directly reachable from the AWX cluster.

Reply:
this is great feedback @AlanCoding. I will just paste in the inventory files for now (non collaspable code-blocks) and introduce reasonings as to `why` for hop nodes.